test/integration/integration_test.go

```diff
@@ -102,6 +113,25 @@ func (i *integrationTest) setup() {
 		}
 		i.setupComplete = true
 	}
+
+	from := flag.CommandLine
+	if vflag := from.Lookup("v"); vflag != nil {
+		fmt.Fprintf(os.Stdout, "integration_test.go: found -v flag from starter bash script with initial glog level %d\n", getLogLevel())
```

Comment:
Actually, seem to be hitting a bit of a chicken / egg situation with the first print.
Neither
glog.V(0).Infof("integration_test.go: found -v flag from starter bash script with initial glog level %d", getLogLevel())
no glog.Infof ....
print that first message.  However, when I then set the -v param, the glogs work.

Would you rather I just not print until I set the logging level, or can I use fmt.Fprintf for that first print?